WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1537,15 +1695,11 @@ static GSourceFuncs _handlerIntervention =
                     _context,
                     [](gpointer customdata) -> gboolean {
                         WebKitImplementation* object = static_cast<WebKitImplementation*>(customdata);
-
-                        if (object->_config.LoadBlankPageOnSuspendEnabled.Value()) {
-                            const char kBlankURL[] = "about:blank";
```

Comment:
We do use this, keep it as it is. Don't enable it if you don't need it.

Reply:
Agrre, do think this is good functionality, but I thought we kept it ?

Reply:
Good find indeed. Intention was to keep this, but with code changes on both repos it was accidentally removed when merging. Will of course bring this back